server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -2503,6 +2504,52 @@ public TsFileManagement getTsFileManagement() {
     return tsFileManagement;
   }
 
+
+  public void insert(InsertRowsOfOneDevicePlan insertRowsOfOneDevicePlan)
+      throws WriteProcessException {
+//    for (InsertRowPlan plan : insertRowsOfOneDevicePlan.getRowPlans()) {
+//      insert(plan);
+//    }
```

Comment:
Remove these?

Reply:
It's removed. Thank you for the suggestion!